Terraform/ArkCase-FOIA/ec2.tf

```diff
@@ -0,0 +1,19 @@
+data "terraform_remote_state" "main" {
+  backend = "local"
+config = {
+    path = "../terraform-vpc/terraform.tfstate"
+    }
+}
+
+resource "aws_instance" "web" {
+
+    ami = "${data.aws_ami.arkcase.id}"
+    instance_type = "t3.xlarge"
+    key_name = "foia-hud-arkcase-demo"
```

Comment:
Why do the other ec2.tf files comment out `key_name`, but this one has it enabled?

Reply:
out.terraform should be added to .gitignore, and the key_name should be uncommented on the other ec2 files, since we will use it for auth.